libraries/apollo-api/api/apollo-api.api

```diff
@@ -864,7 +864,6 @@ public final class com/apollographql/apollo3/api/http/ByteStringHttpBody : com/a
 public final class com/apollographql/apollo3/api/http/DefaultHttpRequestComposer : com/apollographql/apollo3/api/http/HttpRequestComposer {
 	public static final field Companion Lcom/apollographql/apollo3/api/http/DefaultHttpRequestComposer$Companion;
 	public static final field HEADER_APOLLO_OPERATION_ID Ljava/lang/String;
-	public static final field HEADER_APOLLO_OPERATION_NAME Ljava/lang/String;
```

Comment:
FWIW, I'm team "drop const" in public APIs now. If a newer version of the lib is pulled during dependency resolution, I want  the new value of constants to stay in sync, it'd be quite surprising else.